Task3/Task3_Nikolay_Varadinov/src/python/car_wash/libPyPool/utils.py

```diff
@@ -124,8 +124,8 @@ def push(self, conn) -> str:
 
```

Comment:
Please remove this file and main from review and submit a code. Next review should contain your pool of connection